signer/signer.go

```diff
@@ -94,6 +97,13 @@ func (s *genericSigner) Sign(ctx context.Context, desc ocispec.Descriptor, opts
 	if opts.ExpiryDuration != 0 {
 		signReq.Expiry = signReq.SigningTime.Add(opts.ExpiryDuration)
 	}
+	logger.Debugf("sign request:")
+	logger.Debugf("  contentType: %v", signReq.Payload.ContentType)
+	logger.Debugf("  content: %s", string(signReq.Payload.Content))
+	logger.Debugf("  signingTime: %v", signReq.SigningTime)
+	logger.Debugf("  expiry: %v", signReq.Expiry)
+	logger.Debugf("  signingScheme: %v", signReq.SigningScheme)
+	logger.Debugf("  signingAgent: %v", signReq.SigningAgent)
```

Comment:
```suggestion
	logger.Debugf("  content type:   %v", signReq.Payload.ContentType)
	logger.Debugf("  content:        %s", string(signReq.Payload.Content))
	logger.Debugf("  signing time:   %v", signReq.SigningTime)
	logger.Debugf("  expiry:         %v", signReq.Expiry)
	logger.Debugf("  signing scheme: %v", signReq.SigningScheme)
	logger.Debugf("  signing agent:  %v", signReq.SigningAgent)
```

Reply:
If upper cases are adopted, 
```suggestion
	logger.Debugf("  ContentType:   %v", signReq.Payload.ContentType)
	logger.Debugf("  Content:       %s", string(signReq.Payload.Content))
	logger.Debugf("  SigningTime:   %v", signReq.SigningTime)
	logger.Debugf("  Expiry:        %v", signReq.Expiry)
	logger.Debugf("  SigningScheme: %v", signReq.SigningScheme)
	logger.Debugf("  SigningAgent:  %v", signReq.SigningAgent)
```

Reply:
Updated.